### PR TITLE
feat: feedback mechanism

### LINKED ISSUE #130: Add feedback mechanism
We need a way to allow non-expert users send bug-reports, feature requests (or praise :-). At the moment this is only doable via Github issues. 

However, the problem is that our user base consists mainly of non tech-savvy people, we can't expect them to be programmers, have a Github account, and so on.

Ideally, this is a low-friction mechanism that allows one to provide feedback without creating an account anywhere, or starting another program (e.g., an email client, a chat client, etc.).

### PR DESCRIPTION
**Pull request by grigorebudac**
Close #130 

Feedback mechanism sidebar preview
Feedback mechanism preview

### CODE REVIEW

**Comment by iamandrewluca** [comment]
@grigorebudac 
I read the docs in more details, you can use the DSN as authentication for this case. Creating a token will require `project:write` permissions, and putting that token in public may be dangerous.

> Note: Feedback may be submitted with DSN authentication (see auth documentation).
> https://docs.sentry.io/api/projects/submit-user-feedback/

**DSN authentication**
https://docs.sentry.io/api/auth#id3

We already have DSN in `.env` file as `REACT_APP_SENTRY_DSN` (yes, DSN is a full URL)

**Comment by grigorebudac** [comment]
@iamandrewluca Aaaand done 😄

**Comment by roata** [comment]
I tried it out and there's one usability remark: when I click `submit`, there's no positive or negative feedback telling me that the message was sent successfully or that there was an error.

The window just disappears, which I presume means "all is well". It would be better if we reassured the user that their feedback was processed.

**Comment by iamandrewluca** [comment]
I'm trying to submit, but it seems we have to make `event_id` unique. Looking at the docs and `event_id` looks like this `14bad9a2e3774046977a21440ddb39b2`

File: src/components/provide-feedback.dialog.tsx [hunk 1]
```diff
@@ -23,18 +24,19 @@ type FeedbackFormValues = {
...
     props.setOpen(false)
   }
 
-  function handleSubmitFeedback(data: FeedbackFormValues) {
+  async function handleSubmitFeedback(data: FeedbackFormValues) {
     const { REACT_APP_SENTRY_DSN, REACT_APP_SENTRY_ORGANIZATION_SLUG, REACT_APP_SENTRY_PROJECT_SLUG } = process.env
 
     const body = {
-      event_id: 'user-feedback',
+      event_id: Date.now(),
```
Comment (iamandrewluca) [inline src/components/provide-feedback.dialog.tsx:39]: Good one 👍

**Review by iamandrewluca — APPROVED** [review]
LGTM. But for whatever reason no feedback is coming in sentry dashboard, maybe we need to send feedback from real domain. I'll merge this for now. We'll see how this progresses

**Comment by iamandrewluca** [comment]
@roata tried to submit feedback from `https://roataway.md/` but no feedback in sentry dashboard, maybe is delayed by sentry.

**Pull request merged**